marklogic-data-hub-spark-connector/src/main/java/com/marklogic/hub/spark/sql/sources/v2/writer/HubDataWriter.java

```diff
@@ -80,6 +82,8 @@ public WriterCommitMessage commit() {
         bulkInputCaller.awaitCompletion();
         if (writeException != null) {
             logger.info("At least one write failed");
+            logger.info("Additional External Metadata - "+ options.get("additionalexternalmetadata"));
```

Comment:
The story isn't calling for this, so I don't think we should add more logging here. I don't think this logging is solving any problem either. Ernie already knows what the Run ID of the Glue job is; he can use that to find the DHF job document.

Reply:
But if the option "additionalexternalmetadata" is not there in the script, there is no way Ernie will  know the Glue Run id. In that case we might want to add a logger saying what and where he needs to add/change his Glue script.

Reply:
I'm not following - I can see logging the DHF job ID here, because if Ernie doesn't provide the Glue Run ID in the options, then the DHF job ID would be useful in the Glue logs as a way of finding the associated DHF job document - though it won't be that helpful, since it won't have any additional info in it. 

But if Ernie doesn't provide additionalexternalmetadata - well that's his choice. Our documentation can make it clear that that's the way he can add additional data to the job document. Also, we wouldn't Glue-specific logging in a Spark-generic connector.